src/qt/walletcontroller.cpp

```diff
@@ -62,6 +62,20 @@ WalletModel* WalletController::openWallet(const std::string& name, QWidget* pare
     return wallet_model;
 }
 
+void WalletController::closeWallet(WalletModel* wallet_model, QWidget* parent)
+{
+    QMessageBox::StandardButton button = QMessageBox::question(parent, tr("Close wallet"),
+        tr("Are you sure you wish to close wallet %1?").arg(wallet_model->getDisplayName()),
+        QMessageBox::Yes|QMessageBox::Cancel,
+        QMessageBox::Yes);
+    if (button != QMessageBox::Yes) return;
+
```

Comment:
It could be written without `button` variable:
```
    if (QMessageBox::Yes != QMessageBox::question(parent, tr("Close wallet"),
        tr("Are you sure you wish to close wallet %1?").arg(wallet_model->getDisplayName()),
        QMessageBox::Yes|QMessageBox::Cancel,
        QMessageBox::Yes)) return;
```

Also [Qt docs](https://doc.qt.io/qt-5/qmessagebox.html#details) do not recommend use the Static Functions API:

> Using the property-based API is recommended.

> The _Microsoft Windows User Interface Guidelines_ recommend using the application name as the window's title, which means that if you have an informative text in addition to your main text, you must concatenate it to the `text` parameter.

Reply:
Done.